contracts/payloads/IGP101/PayloadIGP101.sol

```diff
@@ -0,0 +1,547 @@
+                    });
+
+                    IFluidDex(USDC_USDT_DEX).updateUserBorrowConfigs(config_);
+                }
+
+                VAULT_FACTORY.setVaultAuth(
+                    wstUSR_USDC__USDC_USDT_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+
+            {
+                address wstUSR_USDC__USDC_USDT_CONCENTRATED_VAULT = getVaultAddress(
+                        135
+                    );
+                address USDC_USDT_CONCENTRATED_DEX = getDexAddress(34);
+
+                {
+                    // Update wstUSR-USDC<>USDC-USDT concentrated vault borrow shares limit
+                    IFluidAdminDex.UserBorrowConfig[]
+                        memory config_ = new IFluidAdminDex.UserBorrowConfig[](
+                            1
+                        );
+                    config_[0] = IFluidAdminDex.UserBorrowConfig({
+                        user: wstUSR_USDC__USDC_USDT_CONCENTRATED_VAULT,
+                        expandPercent: 30 * 1e2, // 20%
+                        expandDuration: 6 hours, // 12 hours
+                        baseDebtCeiling: 2_500_000 * 1e18, // 2.5M shares ($5M)
+                        maxDebtCeiling: 5_000_000 * 1e18 // 5M shares ($10M)
+                    });
+
+                    IFluidDex(USDC_USDT_CONCENTRATED_DEX)
+                        .updateUserBorrowConfigs(config_);
+                }
+
+                VAULT_FACTORY.setVaultAuth(
+                    wstUSR_USDC__USDC_USDT_CONCENTRATED_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+    }
+
+    // @notice Action 5: Update CF, LT, LML and Borrow Cap for Gold Smart Vaults
+    function action5() internal isActionSkippable(5) {
+        {
+            address PAXG_XAUT__USDC_VAULT = getVaultAddress(122);
+            address PAXG_XAUT__USDT_VAULT = getVaultAddress(123);
+            address PAXG_XAUT__GHO_VAULT = getVaultAddress(124);
+
+            uint256 CF = 75 * 1e2;
+            uint256 LT = 80 * 1e2;
+            uint256 LML = 83 * 1e2;
+
+            IFluidVaultT1(PAXG_XAUT__USDC_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(PAXG_XAUT__USDC_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(PAXG_XAUT__USDC_VAULT).updateLiquidationMaxLimit(LML);
+
+            IFluidVaultT1(PAXG_XAUT__USDT_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(PAXG_XAUT__USDT_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(PAXG_XAUT__USDT_VAULT).updateLiquidationMaxLimit(LML);
+
+            IFluidVaultT1(PAXG_XAUT__GHO_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(PAXG_XAUT__GHO_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(PAXG_XAUT__GHO_VAULT).updateLiquidationMaxLimit(LML);
+        }
+
+        {
+            address PAXG_XAUT_DEX = getDexAddress(32);
```

Comment:
can put in separate action. also need to update dex limit at LL, we can make it 5M base